alcs-frontend/src/app/services/application/application-document/application-document.service.ts

```diff
@@ -0,0 +1,52 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { firstValueFrom } from 'rxjs';
+import { environment } from '../../../../environments/environment';
+import { ToastService } from '../../toast/toast.service';
+import { ApplicationDocumentDto } from './application-document.dto';
+
+const MAX_FILE_SIZE = 104857600;
```

Comment:
I would move this to config. I think this is something that should be easily controlled without commit

Reply:
Will do, I was on the fence about it. I can also move the one on the front-end to the environment file.

Reply:
Done!